client/components/checkbox/style.scss

```diff
@@ -0,0 +1,52 @@
+.checkbox {
+	cursor: pointer;
+	display: block;
+	font-size: 14px;
+	margin-bottom: 5px;
+	position: relative;
+	user-select: none;
+}
+
+.checkbox input[type="checkbox"] {
+	@include hide-content-accessibly;
+	border: 0;
+	margin: -1px;
```

Comment:
I'm just curious, not leaving a comment about the code, but why the `-1px` margin here?